assets/src/js/admin/onboarding-wizard/components/select-input/style.scss

```diff
@@ -0,0 +1,48 @@
+// .give-obw-select-input {
```

Comment:
If not used we need to remember to clean this out. Not picky about it right now, as scaffolding, just making a note.

Reply:
Good catch, cleared up this code now. ref: https://github.com/impress-org/givewp/pull/4942/commits/fbbab93ce7c552c66e541a7b5a9ef62c9e04007c